### ISSUE TITLE
New config key for files metadata

### ISSUE THREAD

**Issue by mmattel** [owncloud/docs#4419]
## WHAT Needs to be Documented?
https://github.com/owncloud/core/pull/39126 (Feature/version metadata)
`'file_storage.save_version_author' => true`

## WHERE Does This Need To Be Documented (Link)?
`config-to-docs` run and in user_manual/pages/files/version_control.adoc

## WHY Should This Change Be Made?
Completeness of docs

## (Optional) What Type Of Content Change Is This? 
- [x] New Content Addition
- [ ] Old Content Deprecation
- [ ] Existing Content Simplification
- [ ] Bug Fix to Existing Content

## (Optional) Which Manual Does This Relate To?
- [x] Admin Manual
- [ ] Developer Manual
- [x] User Manual
- [ ] Android
- [ ] iOS
- [ ] Branded Clients
- [ ] Desktop Client
- [ ] Other

@JanAckermann

**Comment by mmattel** [comment]
After some testing to create screenshots:

```
user1 creates testfile.txt - no versions shown
admin changes testfile.txt - version admin
user1 - sees no versions shown
user1 changes testfile.txt - version user (with a timestamp 3 minuten ago)
admin - sees version admin
````
@JanAckermann @jnweiger any tip for my understanding

**Comment by mmattel** [comment]
> I am not sure that the doc screenshots... can be done until we have code that behaves as specified.

Correct, as long that is not fixed, I will not continue on how to implement this in docs. Thanks for linking this to the followup core PR. Just wanted to do the same.

@IljaN added you to the docs loop

**Comment by jnweiger** [comment]
Tested that feature in 10.9.0 beta1 -- seems to work correctly for me.

* The unusual use of  'author' might need clarification. One could expect that the author is the original creator of the document, and later editors are called contributor or similar.
  In this context, everybody is called an 'author'. They 'author' a new version, even if they just delete a line.

* trashbin delete / restore behaviour might change due to https://github.com/owncloud/core/issues/39571

**Comment by phil-davis** [comment]
"The unusual use of 'author' might need clarification."

The tricky part with the words is that, I guess, the oldest version (the one that first existed) has an "author" and then each version after that has a person who changed the file. They may not have even "edited" it. If it is a file that some local application "changed for them" then they won't have "opened the file in Word or Excel or Powerpoint or an image/video app and edited it". The file will have changed and been synced up to the cloud storage without them really being aware they the are somehow the "author" or "editor".

When the file  has been upload and then changed 3 times there is:
- oldest version (3rd version) has an "author"
- 2nd oldest version has a "changer"
- most recent version in the version history has a "changer"
- the current file has a "changer" (what do we even call the current file - for me it is "the latest version of the file", but that might create confusion because there is also "the most recent version in the version history" - some people might look at the version history and say that " the most recent version in the version history" is what they would describe as " the latest version of the file".

Words are hard!

**Comment by DeepDiver1975** [comment]
This repository is being archived and is no longer maintained. Closing this as part of archival. Thank you for your contribution.
